copier.yml

```
@@ -34,6 +34,7 @@ node_version:
 
 
 
+
```

YAMLlint (1.37.1)

[error] too many blank lines (4 > 2)

(empty-lines)